runtime/pom.xml

```diff
@@ -44,36 +44,43 @@
     <finalName>runtime</finalName>
     <plugins>
       <plugin>
-        <groupId>org.wildfly.swarm</groupId>
-        <artifactId>wildfly-swarm-plugin</artifactId>
-        <version>${wildfly.swarm.version}</version>
-        <configuration>
-          <mainClass>com.redhat.ipaas.runtime.Main</mainClass>
-          <jvmArguments>
-            <jvmArgument>-server</jvmArgument>
-            <jvmArgument>-Xms256m</jvmArgument>
-            <jvmArgument>-Xmx512m</jvmArgument>
-          </jvmArguments>
```

Comment:
You're funny. We know the are going into battle but we don't know what type of bullets will be used, so we don't know what the thickness of the armor needs to be. So let's use no armor until we find that out. OK grrrreat.